libraries/consul_service.rb

```diff
@@ -46,6 +46,14 @@ class ConsulService < Chef::Resource
       # The location of the Consul executable.
       # @return [String]
       attribute(:program, kind_of: String, default: '/usr/local/bin/consul')
+      # @!attribute restart_on_update
+      # Whether to restart the service on updates
+      # @return [Boolean]
+      attribute(:restart_on_update, kind_of: [TrueClass, FalseClass], default: lazy { node['consul']['service']['restart_on_update'] })
```

Comment:
Can we default this to false? This will still work as expected (using node attribute) because of [how we wire attributes into the service resource](https://github.com/johnbellone/consul-cookbook/blob/master/recipes/default.rb#L70-L72).